src/scss/components/_sortable-table.scss

```diff
@@ -0,0 +1,20 @@
+[aria-sort="ascending"] a,
```

Comment:
I think you'll have to add some overflow CSS styles for when the screen is mobile. 
![Screenshot 2025-07-01 at 11 56 25](https://github.com/user-attachments/assets/a5a67509-6451-4da4-b15a-dbcac51a0d1f)